src/components/views/dialogs/security/AccessSecretStorageDialog.tsx

```diff
@@ -351,14 +356,14 @@ export default class AccessSecretStorageDialog extends React.PureComponent<IProp
 
                     <form className="mx_AccessSecretStorageDialog_primaryContainer" onSubmit={this.onPassPhraseNext}>
                         <Field
+                            inputRef={this.inputRef}
                             id="mx_passPhraseInput"
                             className="mx_AccessSecretStorageDialog_passPhraseInput"
                             type="password"
                             label={_t("Security Phrase")}
                             value={this.state.passPhrase}
                             onChange={this.onPassPhraseChange}
                             autoFocus={true}
-                            autoComplete="new-password"
```

Comment:
How is this an a11y improvement? Given most password managers only tend to store 1 pass per-website this will risk it being auto-filled with your account password

Reply:
You are right, it will autofill the users password into the field.
We had the internall suggestion to remove the autocomplete attribute to comply more with the german a11y law (BITV)
The reason for this was, that new-password is missleading the user to create a new password and not just enter the security phrase.
I will revoke this change, since autofilling the password is worse a11y.

Reply:
A quick question. Can you answer me if saving the secure phrase in the browser is safe and could be an option?
I had the idea to set a hidden `<input name="_t('Secure Phrase')">` field along with the secure phrase input to give the phrase a name.
So the browser can store the phrase behind the 'Secure Phrase' key and does not try to overwrite the users password.
This allows to store mutliple passwords for one page.
If this is out of the scope of this pr i will try to ask in the Web Dev Channel.

Reply:
> Can you answer me if saving the secure phrase in the browser is safe and could be an option?

That depends on the individual, people have different security threat models, different integrated password managers, some backed by their system's secure enclave, etc.

I suggest separating the autocomplete tweaks to their own PR as they are more nuanced.